dev/integration_tests/abstract_method_smoke_test/android/app/src/main/kotlin/com/example/abstract_method_smoke_test/MainActivity.kt

```diff
@@ -12,8 +14,22 @@ class MainActivity: FlutterActivity() {
     GeneratedPluginRegistrant.registerWith(this)
 
     // Triggers the Android keyboard, which causes the resize of the Flutter view.
-    // https://github.com/flutter/flutter/issues/40126
+    // We need to wait for the app to complete.
+    MethodChannel(getFlutterView(), "com.example.abstract_method_smoke_test")
```

Comment:
Is this a bug?  Why can't an app show the keyboard before Flutter is ready?

Reply:
Perhaps the keyboard is getting hidden from the fuzzer tapping randomly (and maybe tapping on the text input and then away from it...)